demos/sdk_output_skeleton_2_ghe_staggered/run/baseline_scenario/ghe_dir/sys_params.json

```diff
@@ -82,7 +82,7 @@
         "insulation_thickness": 0.2,
         "insulation_conductivity": 2.3,
         "diameter_ratio": 11,
-        "roughness": 1e-6,
+        "roughness": 1e-06,
```

Comment:
@vtnate why does the format on these keep flip-flopping?

Reply:
Do they? Uuuuuuugh. It's Ruff. I can look into which rule is being hit and turn it off.